### DISCUSSION TITLE
How to implement facebook api conversions with nextjs 13??

### DISCUSSION THREAD

**Discussion by carloscurcino** [vercel/community#2959] [Help]
### Summary

I made a website for a client, using nextjs 13, and he wants to advertise on facebook ads, but for that we need to use the facebook pixel and the conversions API to collect user data. I implemented the pixel following the example of the vercel repository. But I haven't found an easy way to implement Facebook API conversions. I used the following @rivercode/facebook-conversion-api-nextjs package following the instructions in the documentation, but I'm not sure it's working, it doesn't seem to be.

There's any easy way implement it in next or maybe some integration in vercel deployment?? Someone can help me??

### Steps to Reproduce

n

**Comment by smaeda-ks** [comment]
@carloscurcino have you opened a new issue on https://github.com/RivercodeAB/facebook-conversion-api-nextjs ?

Since this is a third-party library that we don't own, I recommend asking the question to the author.
